entregas/PerozoDiego/src/PiedraPapelTijera.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+
+public class PiedraPapelTijera {
+    public static void main(String[] args) {
+
+        Scanner input = new Scanner(System.in);
+
+        int EleccionJugador;
```

Comment:
Los nombres de variables empiezan en minúsculas y usan camel case en caso de constar de más de una palabra

```suggestion
        int eleccionJugador;
```